x-pack/plugins/rollup/public/crud_app/sections/job_list/job_list.js

```diff
@@ -127,6 +127,38 @@ export class JobListUi extends Component {
     );
   }
 
+  renderError(error) {
+    // We can safely depend upon the shape of this error coming from Angular $http, because we
+    // handle unexpected error shapes in the API action.
+    const {
+      statusCode,
+      error: errorString,
+    } = error.data;
+
+    const { intl } = this.props;
+    const title = intl.formatMessage({
+      id: 'xpack.rollupJobs.jobList.errorTitle',
+      defaultMessage: 'Error loading rollup jobs',
+    });
+    return (
+      <Fragment>
+        {this.getHeaderSection()}
+        <EuiSpacer size="m" />
+        <EuiCallOut
+          title={title}
+          color="danger"
+          iconType="alert"
+        >
+          <FormattedMessage
```

Comment:
There is nothing to translate. So we can leave as it is.
```suggestion
          {statusCode} {errorString}
```